### PR TITLE
Compact brief

### PR DESCRIPTION
**Pull request by jrchudy**
This PR addresses the following:

- using the `compact/brief` context for references in record2 for the related tables (#570)
- a link after the related tables that goes to recordset when there are more than 5 values
- a link after markdown content as well
- created DOM structure for different display types in record2 (#580)

I don't have a link to an entity that has a related table with more than 5 values associated. I did write a test that makes sure that is working properly though.

### CODE REVIEW

File: record-two/index.html.in [hunk 1]
```diff
@@ -35,7 +35,18 @@
...
+                <!-- for different elements inside the accordion -->
+                <div ng-switch on="relatedRef.display.type">
+                    <div ng-switch-when="markdown">
+                        <span ng-bind-html="page.content"></span>
+                        <span ng-if="page.hasNext" ng-controller="RecordController as ctrl" ng-click="ctrl.toRecordSet(relatedRef)">Click here to access more records</span>
+                    </div>
+                    <!-- ng-switch-when="table" is also the default case -->
+                    <div ng-switch-default>
+                        <record-table class="related-table" id="rt-{{relatedRef.displayname}}" vm="tableModels[$index]"></record-table>
+                        <a ng-controller="RecordController as ctrl" ng-click="ctrl.toRecordSet(relatedRef)">Click here to access more records</a>
```
Comment (shinyichen) [inline record-two/index.html.in]: Did you intentionally leave out `ng-if="page.hasNext"` here?

Comment (jrchudy) [inline record-two/index.html.in]: That was for testing. This PR isn't technically ready yet, I still need to flesh out the test cases.

**Comment by jrchudy** [comment]
@shinyichen, this PR is ready for review now.

**Comment by shinyichen** [comment]
I didn't see any problems. Also I tested manually the linking, no problem there.

**Pull request merged**